site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -1681,9 +1681,12 @@ public function getBuildStatusOfGradeable(string $gradeable_id): void {
                 $logs = $this->getBuildLogs($gradeable_id);
 
                 $needle = 'The submitty configuration validator detected the above error in your config.';
+                $errorNeedle = 'ERROR';
                 $haystack = $logs->json['data'][0];
-
-                if (strpos($haystack, $needle) !== false) {
+                if (strpos($haystack, $errorNeedle) !== false) {
+                    $status = false;
+                }
+                elseif (strpos($haystack, $needle) !== false) {
```

Comment:
It would be more clear to use `str_contains()` here.  I think `$errorNeedle` should also just be inlined since the string is so short.